Makefile

```diff
@@ -58,19 +57,17 @@ busybox.tgz:
 	docker save busybox:latest | gzip -c > busybox.tgz
 
 calico-node.tgz:
-	docker pull calico/node:latest
-	docker save calico/node:latest | gzip -c > calico-node.tgz
+	docker pull calico/node:v0.10.0
```

Comment:
Urm, what am I missing here?  If you pin calicoctl, then you automatically pin calico/node and calico/node-libnetwork - doesn't that mean the STs will never use the latest version of the code if we pin?.  Unless we explicitly pass in which libnetwork version to use in `calicoctl node`.